base_rest_pydantic/tests/test_from_params.py

```diff
@@ -22,8 +22,14 @@ class Model1(BaseModel):
 
         self.Model1: BaseModel = Model1
 
-    def _from_params(self, pydantic_cls: Type[BaseModel], params: dict, **kwargs):
-        restapi_pydantic = restapi.PydanticModel(pydantic_cls, **kwargs)
+    def _from_params(
+        self,
+        pydantic_cls: Type[BaseModel],
```

Comment:
@petrus-v Something is wrong here. The second arg is the service not a BaseModel type.

Reply:
I don't get it :-/... may be we should rename this method in unit test that call the from_params method on the PydanticModel for the given BaseModel. 

Service is moked bellow before calling from_params !

Do I missed something ?

Reply:
we could call it `_from_params_call` ?

Reply:
@petrus-v My bad, we are into the tests module....

Why do you need to pass the restapi class to the `_from_params' method. We don't need it. IMO we've 2 possibilities:
* `restapi_pydantic = restapi.PydanticModelList if isinstance(params, list) else restapi.PydanticModel`
* Add an explicit parameter to the method to specify that the params are a list...

Reply:
I'm fine with your first suggestion, providing the restapi class as parameter was a way for me to explicitly test what I wanted to test here !

I'll change that !